frontend/Dockerfile

```diff
@@ -1,21 +1,12 @@
-# pclubiitk/student-search:frontend
-FROM nginx
-RUN apt-get update
-RUN apt-get -y install gnupg && apt-get -y install curl && \
-    curl -sL https://deb.nodesource.com/setup_6.x | bash - && \
-    apt-get install -y nodejs
-
-# Install yarn and ng-cli
-RUN npm install -g yarn @angular/cli
-
-# Cached layer for node_modules
-COPY package.json /tmp/package.json
-ADD yarn.lock /tmp/yarn.lock
-RUN cd /tmp && yarn install
-RUN mkdir -p /src && cp -a /tmp/node_modules /src/
-
-WORKDIR /src
-ADD . /src
-RUN cd /src && yarn build:deploy
-
+### STAGE 1: Build ###
+FROM node:6.16-alpine AS build
+WORKDIR /usr/src/app
+COPY package.json ./
+RUN npm install
```

Comment:
Why did we get rid of yarn? 😢 I used to like it a lot. Way faster than npm. Plus it had its own yarnfile which won't be respected for the build if you use npm?

Reply:
The exact dependencies are now respected by npm's `package-lock.json` file. So I don't think this should be an issue.

Reply:
But it won't respect the exact dependencies of the existing yarn.lock file right. It needs to be once built with npm and then the npm package-lock has to be committed.

Reply:
Hmm, you're right... unless one uses npm 7.0 which respects `yarn.lock` versions as well.